dsaas-services/f8-forker.yaml

```diff
@@ -1,5 +1,5 @@
 services:
-- hash: master
+- hash: 62562f02d25e13756913c1a02bbf1916cda662b0
   name: fabric8-forker
```

Comment:
Yeah..problem with this is that we have no simple way to verify the image has been pushed..Jenkins does not have the information. We can workaround it in the service.yaml by adding something like "skip_update" or just check if there is "master" in hash and if yes, don't update it..or we can put some exception in jenkins-update.py